Dependencies/napi/napi-direct/CMakeLists.txt

```
@@ -1,14 +1,20 @@
+if(NAPI_JAVASCRIPT_ENGINE STREQUAL "V8")
+    set(ENGINE_FILE_POSTFIX "v8")
+else()
+    set(ENGINE_FILE_POSTFIX ${NAPI_JAVASCRIPT_ENGINE})
+endif()
+
```

```suggestion
string(TOLOWER NAPI_JAVASCRIPT_ENGINE ENGINE_FILE_POSTFIX)
```

Then update the local file for JSC to lower case.